airbyte-integrations/connectors/source-cockroachdb/src/test/java/io/airbyte/integrations/source/cockroachdb/CockroachDbJdbcSourceAcceptanceTest.java

```diff
@@ -212,8 +212,8 @@ protected void testDiscoverWithNullableCursorFields() throws Exception {
   void testCheckFailure() throws Exception {
     ((ObjectNode) config).put(JdbcUtils.PASSWORD_KEY, "fake");
     ((ObjectNode) config).put(JdbcUtils.USERNAME_KEY, "fake");
-    final AirbyteConnectionStatus actual = source.check(config);
```

Comment:
I don't understand this change. Did we just rename `actual` variable to `status`? If so, this is not a functional change and appears to be a distraction.